README.md

```diff
@@ -167,14 +160,14 @@ disease:
 
 then the `c01` would be mapped to both `output_class_c` and `disease`. This scheme allows many-to-many mappings.
 
-Records assigned to classes that are not mapped will be dropped, therefore will not exist in the output dataset. Default mapping behavior, that occurs when class mapper is unspecified, maps classes one to one, ex: `c01 -> c01` and assumes that `h01-h07` and `b02` are healthy classes.
+Records assigned to class that are not mapped will be dropped, therefore will not exist in the output dataset. Default mapping behavior, that occurs when class mapper is unspecified, maps classes one to one, ex: `c01 -> c01` and assumes that `h01-h07` and `b02` are healthy classes.
 
-##### Healthy flag
-In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as healthy. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
-- if the mask belongs to a healthy class **and** segmentation is not binary, the color is white.
-- otherwise, the color is black.
+##### Positive flag
+In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as positive. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
+- if  segmentation is binary **and**, the mask belongs to a none positive class the color is black.
+- otherwise, the color is white.
 
-If there is a valid mask file that belongs to a class labeled as healthy and the segmentation is not binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
+If there is a valid mask file that belongs to a class labeled as not positive, and the segmentation is binary, then the colors in the mask will get reversed. White pixels will switch places with the black pixels.
 
 ##### Mapping example
```

Comment:
Section on output mask and frame naming would be needed. For ers it's `"patient_id}_{data_dir_basename}_{frame_name}.png"` and for hkvs we just copy the filename.

I think then this should be shown in mapping scheme too

Reply:
Done